modules/doctor/lfs.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package doctor
+
+import (
+	"context"
+
+	"code.gitea.io/gitea/modules/log"
+	"code.gitea.io/gitea/services/repository"
+)
+
+func init() {
+	Register(&Check{
+		Title:                      "Garbage collect LFS",
+		Name:                       "gc-lfs",
+		IsDefault:                  false,
```

Comment:
Maybe it should be true if LFS is enabled?

Reply:
well ... it could take a very long time to run this. So I don't think it's appropriate as a standard check

Reply:
plus LFS can be disabled - which will cause this check to return an error.

Reply:
> 

I mean 
```go
if setting.LFS.StartServer {
      Register(&Check{
		Title:                      "Garbage collect LFS",
		Name:                       "gc-lfs",
		IsDefault:                  true,
      })
}
```

Reply:
@lunny I'm not sure, I think the `stroage-*` checks do something like that, but they are not default.

<img width="1297" alt="image" src="https://user-images.githubusercontent.com/9418365/206622651-71614ea0-b74c-4d12-83d1-39f11029da80.png">

Reply:
Sorry, I mean we need a cron task to check that by default. Not doctor. Maybe that will be another PR @zeripath ?

Reply:
For a cron task we need to closely review the order of LFS uploads etc. and seriously think about the issue of concurrency.

We should also add an last_updated to the LFSMetaObject to allow for an incremental GC.